spec/presenters/api/v2/measures/measure_presenter_spec.rb

```diff
@@ -10,6 +10,38 @@
     end
   end
 
+  context 'when measure has a generating regulation' do
+    let(:measure) { create(:measure, :with_base_regulation) }
+
+    describe '#measure_generating_legal_act' do
+      it 'returns the MeasureLegalActPresenter' do
+        expect(presenter.measure_generating_legal_act).to be_instance_of(Api::V2::Measures::MeasureLegalActPresenter)
```

Comment:
This us a useful spec and should be kept but it also needs to be checking the object returned is the expected one? eg comparing the id as the `measure_generating_legal_act_id` spec does

Reply:
Hi @jebw ,
I think this is covered by the following spec: https://github.com/trade-tariff/trade-tariff-backend/pull/843/files#diff-131c0beeeee7f7d07f72f9baf49e3c3e1e46d6e74cc1dff07891103108ac602dR23

Do you agree?